libc/include/llvm-libc-types/ssize_t.h

```diff
@@ -9,6 +9,12 @@
 #ifndef LLVM_LIBC_TYPES_SSIZE_T_H
 #define LLVM_LIBC_TYPES_SSIZE_T_H
 
+// https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types
+#if __has_include(<BaseTsd.h>)
```

Comment:
`__has_include` isn't the appropriate way to check if this is a windows target. I'd recommend doing `#ifdef WIN32` instead.